test/CMakeLists.txt

```diff
@@ -515,6 +519,11 @@ add_umf_test(
     SRCS ipcAPI.cpp ${BA_SOURCES_FOR_TEST}
     LIBS ${UMF_UTILS_FOR_TEST})
 
+add_umf_test(
+    NAME ipc_max_opened_limit
+    SRCS ipcAPI.cpp ${BA_SOURCES_FOR_TEST}
+    LIBS ${UMF_UTILS_FOR_TEST} ENVS "UMF_MAX_OPENED_IPC_HANDLES=10")
```

Comment:
nit: please extend also 'add_umf_test' in /.cmake-format

Reply:
Done